src/components/document/document-workspace.tsx

```diff
@@ -399,4 +423,22 @@ export class DocumentWorkspaceComponent extends BaseComponent<IProps> {
       return this.stores.documents.getDocument(documentKey);
     }
   }
+
+  private toggleExpandWorkspace = () => {
+    const { ui } = this.stores;
+    if (ui.dividerPosition === 100) {
```

Comment:
Should any of these by constants (0, 50, 100)?  Maybe that's not needed.

Reply:
I was on the fence earlier but I think it'd probably help. Something like `kDividerMin`, `kDividerHalf`, `kDividerMax` perhaps? When the final implementation is in place `kDividerHalf` will go away but the others will still be useful.